src/main/java/configuration/env.java

```diff
@@ -0,0 +1,19 @@
+package configuration;
+
+import java.io.FileInputStream;
+import java.io.InputStream;
+import java.util.Properties;
+
+public class env {
+    public static final Properties CONFIG = read();
+    private static final String CONFIG_PATH = "properties.properties";
```

Comment:
properties.properties is not a good name. you need to rename it like config.properties, salesforce.properties and so on.